bin/wasm-node/javascript/src/compat-browser.js

```diff
@@ -21,5 +21,5 @@ export const net = null;
 export const Worker = typeof window != 'undefined' ? window.Worker : null;
 export const workerOnMessage = (worker, callback) => { worker.onmessage = (event) => callback(event.data) };
 export const workerOnError = (worker, callback) => { worker.onerror = callback; };  // TODO: unclear if the parameter of the callback is same as with NodeJS
-export const postMessage = (msg) => self.postMessage(msg);
+export const postMessage = (msg, chain_index) => self.postMessage(msg, chain_index);
```

Comment:
```suggestion
export const postMessage = (msg) => self.postMessage(msg);
```

This code is here to mimic https://developer.mozilla.org/fr/docs/Web/API/Window/postMessage